modules/nw-sriov-configuring-operator.adoc

```diff
@@ -33,10 +33,10 @@ The `SriovOperatorConfig` object provides several fields for configuring the ope
 The Network Resources Injector is a Kubernetes Dynamic Admission Controller
 application. It provides the following capabilities:
 
-* Mutation of resource requests and limits in `Pod` specification to add an SR-IOV resource name according to an SR-IOV network attachment definition annotation.
-* Mutation of `Pod` specifications with downward API volume to expose pod annotations and labels to the running container as files under the `/etc/podnetinfo` path.
+* Mutation of resource requests and limits in a pod specification to add an SR-IOV resource name according to an SR-IOV network attachment definition annotation.
+* Mutation of a pod specification with a Downward API volume to expose pod annotations, labels, and hugepages requests and limits. Containers that run in the pod can access the exposed information as files under the `/etc/podnetinfo` path.
 
-By default the Network Resources Injector is enabled by the SR-IOV operator and runs as a daemon set on all master nodes. The following is an example of Network Resources Injector pods running in a cluster with three master nodes:
+By default, the Network Resources Injector is enabled by the SR-IOV Network Operator and runs as a daemon set on all master nodes. The following is an example of Network Resources Injector pods running in a cluster with three master nodes:
 
```

Comment:
Would it make more sense to add the "Hugepages resource injection for Downward API" section here instead with the app-netutil? It's the Network Resources Injector doing the injection. app-netutil is the consumer.
include::modules/nw-sriov-hugepages.adoc[leveloffset=+2]

Reply:
I was tempted too.  However, I think it's really that this conceptual info belongs on the same "About SR-IOV" page that I used, or maybe an as-yet-unwritten page for "About the SR-IOV Network Operator."

For now, my preference is to leave as-is.  I hope to gain enough experience in this area that making larger-scale revisions isn't so daunting.